XRTK-Core/Packages/com.xrtk.core/package.json

```diff
@@ -10,7 +10,7 @@
     "Mixed Reality",
     "DI"
   ],
-  "version": "0.1.20",
+  "version": "0.1.21",
```

Comment:
A little premature updating this for a feature branch :D

Reply:
Nope. Always increment for breaking changes.

Reply:
Doctors also used to always just saw legs off at the first sign of infection.   Never say "Always"